mTag-SDK/API.swift

```diff
@@ -208,16 +214,31 @@ open class API: NSObject {
   }
 
   /**
-   Handles the actual registration of the interaction, and passes the parsed response to the delegate.
+   Handles the case where we need to pass an unmutated URL to the interactions route.
+   */
+  class func registerInteraction(withUrl url: String) {
+    let params : [String: String] = [
+      "url": url
+    ]
+    return registerInteraction(withParams: params)
+  }
+
+  /**
+   Handles formatting the interaction's request params for most URL formats.
    */
   class func registerInteraction(withTagId tagId: String, andParams params: [String: String]) {
     _log("Hitting Interactions route", level: "DEBUG")
     var mutableParams:[String: String] = params
     mutableParams["tech"] = "n"
     mutableParams["tag_id"] = tagId
+    return registerInteraction(withParams: mutableParams)
+  }
 
-    _log("Params: \(mutableParams)", level: "DEBUG")
-
+  /**
+   Handles formatting request headers and submitting the actual interaction to the mTag API.  Passes response along to proper delegate.
+   */
+  class func registerInteraction(withParams params: [String: String]) {
+    _log("registering interaction with params: \(params)", level: "DEBUG")
```

Comment:
Where do these logs go? CloudWatch?

Reply:
Nowhere.  The SDK has no integration with cloudwatch or otherwise since it is meant to be public facing, so they are strictly for local debug

Reply:
The actual `_log` func is just a pretty wrapper around a `print` call